[1.12.2] Just released `CraftTweaker2-1.12-4.1.20.664.jar` immedeately crashing on startup

### Issue description

While prepping a release for my modpack [Rotten Economy](https://www.curseforge.com/minecraft/modpacks/rotten-economy) I updated to the just released 1.12.2 version `CraftTweaker2-1.12-4.1.20.664.jar` and could no longer start minecraft as it crashes almost immediately with a stack trace pointing towards CraftTweaker. 

## Stack Trace Snippet
The full trace is in the attached gist link but here is a quick peek:
```
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from CraftTweaker2 (crafttweaker)
 Caused by: java.lang.ExceptionInInitializerError
     at crafttweaker.CrafttweakerImplementationAPI.init(CrafttweakerImplementationAPI.java:72)
     at crafttweaker.mc1120.CraftTweaker.onConstruction(CraftTweaker.java:110)
     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
...
 Caused by: java.lang.RuntimeException: already has a getter
     at stanhebben.zenscript.type.natives.ZenNativeMember.setGetter(ZenNativeMember.java:34)
     at stanhebben.zenscript.type.ZenTypeNative.complete(ZenTypeNative.java:142)
     at stanhebben.zenscript.compiler.TypeRegistry.getClassType(TypeRegistry.java:52)
     at stanhebben.zenscript.compiler.TypeRegistry.getType(TypeRegistry.java:77)
     at stanhebben.zenscript.type.natives.JavaMethod.<init>(JavaMethod.java:37)
```

## Versions
| md5sum | filename | name |
| --- | --- | --- |
| `71c750eb05c4cdcd78d9ed30ac0f8a51` | `CraftTweaker2-1.12-4.1.20.664.jar` | "new", "just released", "crashing" |
| `917948d6b8210b226744372f6f61922c` | `CraftTweaker2-1.12-4.1.20.663.jar` | "old", "previous", "works good" |

Thanks much for including the new IBlockDefinition getters! Excuted to use them soon! Cheers!

### Steps to reproduce

n/a

### Script used

n/a

### The crafttweaker.log file

n/a

### Minecraft version

1.12

### Forge version

forge-1.12.2-14.23.5.2855.jar

### CraftTweaker version

CraftTweaker2-1.12-4.1.20.664.jar

### Other relevant information

n/a

### The latest.log file

https://gist.github.com/ubergarm/4f31230c94fe0242dfca62820008b25d

--- a/CraftTweaker2-API/src/main/java/crafttweaker/api/block/IBlockDefinition.java
+++ b/CraftTweaker2-API/src/main/java/crafttweaker/api/block/IBlockDefinition.java
@@ -27,7 +27,7 @@ public interface IBlockDefinition {
     @ZenSetter("lightOpacity")
     void setLightOpacity(int lightOpacity);
 
-    @ZenGetter("lightLevel")
+    @ZenGetter("lightOpacity")
     @ZenMethod
     default float getLightOpacity() {
         CraftTweakerAPI.logError(

